src/components/UserProfile/AddNewUserProfile/UserProfileAdd.jsx

```diff
@@ -40,7 +40,7 @@ import 'react-phone-input-2/lib/style.css'
 
 import classnames from 'classnames'
 import TimeZoneDropDown from '../TimeZoneDropDown'
-
+var patt = '';
```

Comment:
I dont see any reference of this global variable and if you really need, Can you please update it to "let pattern"?

Reply:
changed 'part' to a const and removed duplicate occurrences